### ISSUE TITLE
can't create no flag commands.

### ISSUE THREAD

**Issue by wh1tecat-nya** [akito0107/catacli#13]
I tried create simple command that uses only arguments (like a file path).
But if do not assignment `flag` property for `makeCommand` function 2nd arguments, get an Error: `TypeError: t2 is not a function`

I did investigation for this.

`makeCommand` option of `flag` is Partial type in`type CommandSpec`.
https://github.com/akito0107/catacli/blob/14d54bdf9c4fbcdd98c9953504e163be8bd9db3b/src/command.ts#L5-L26

But inside of `makeCommand` function, `spec.flag` is calling as a function.
https://github.com/akito0107/catacli/blob/14d54bdf9c4fbcdd98c9953504e163be8bd9db3b/src/command.ts#L45-L54

In short, if do not assignment `flag` property, `spec.flag` is undefined but called as as function.
then get an Error: `TypeError: t2 is not a function`.

that is correct?
and if correct this, may i try create a pull request?

**Comment by akito0107** [comment]
@wh1tecat-nya 
Thank you for reporting.

> that is correct?

I confirmed this issue and this is a bug. I didn't care about `no flag` scenario, sorry.

> and if correct this, may i try create a pull request?

PRs are very welcome and thank you for contribution !
